### PR TITLE
jormungandr: check the {sn}_speed for correctness

### PR DESCRIPTION
**Pull request by TeXitoi**

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/parsers.py [hunk 1]
```diff
@@ -36,6 +36,13 @@ def depth_argument(value, name):
     return int(value)
 
 
+def speed_value(value, name):
+    if float(value) <= 0:
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/parsers.py]: minor: the name may be `positive_float` to be more generic :)

**Comment by xlqian** [comment]
ok for me

File: source/jormungandr/jormungandr/interfaces/parsers.py (hunk 1 above)
Comment (francois-vincent) [inline source/jormungandr/jormungandr/interfaces/parsers.py]: avoid double conversion:
`value = float(value)`

better:

``` python
message = "The {} argument must be float > 0, you gave : {}"
try:
    value = float(value)
except ValueError:
    raise ValueError(message.format(name, value))
if value <= 0:
    raise ValueError(message.format(name, value))
```

and add 'x' in the unitary test case list

Comment (TeXitoi) [inline source/jormungandr/jormungandr/interfaces/parsers.py]: I can't do that, it'll change the message if the conversion fails.

**Pull request merged**